build/webpack/webpack.extension.node.config.js

```diff
@@ -38,12 +38,16 @@ const config = {
             {
                 test: /\.ts$/,
                 use: [
-                    {
-                        loader: 'vscode-nls-dev/lib/webpack-loader',
-                        options: {
-                            base: constants.ExtensionRootDir
-                        }
-                    },
+                    ...(process.env.BUILD_WITH_VSCODE_NLS
```

Comment:
Is this name ok? What other name would be better? How can we provide this to our CI pipeline?

Reply:
Well there may be environment variables already active in the azure-pipeline stuff. But if not you can add one into our versions.

Reply:
For now I'm thinking on setting this environment variable in a `package:nls` npm script, and using that instead of `package` as the default package command.

Reply:
For now I have changed how our build `.yml` files work. Instead of `gulp prePublishBundle` they will call `npm run prePublishBundleNLS`, which injects the environment variable with `cross-env`. I think it's a very versatile approach with minimal maintenance!